test/lint/lint-all.sh

```diff
@@ -16,21 +16,25 @@ set -u
 SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
 LINTALL=$(basename "${BASH_SOURCE[0]}")
 
-for f in "${SCRIPTDIR}"/lint-*.sh; do
-  if [ "$(basename "$f")" != "$LINTALL" ]; then
-    if ! "$f"; then
-      echo "^---- failure generated from $f"
-      exit 1
-    fi
-  fi
-done
+# TODO: Replace with "${SCRIPTDIR}"/lint-*.sh
```

Comment:
I would rather not follow up on this TODO, I prefer explicit stuff, not grep magic...

Reply:
Makes sense. Removed.